docs/self-hosted-appcircle/configure-server/integrations-and-access/integration.md

```diff
@@ -43,6 +44,7 @@ Explanation of each key
 |ssl | If set to true, use SSL to connect |
 |auth | If set to true, attempt to authenticate the user using the AUTH command. |
 |starttls | If set to true, enables the use of the STARTTLS command |
+|verifyCertificate | true by default. If set to false, disables validation of the SMTP server's SSL certificate. Shouldn't be disabled in production.  |
```

Comment:
The first sentence should be updated. The spelling is not proper. 
The other options start with "If set to..."
Recommended description would be: "If set to false, disables validation of the SMTP server's SSL certificate. The value is true by default. Shouldn't be disabled in production environments.

Reply:
➕ We can identify the possible values as code formatting like `true` or `false` so that it can be read easily in the text.